packages/api-sdk/src/resources/apiKey.ts

```diff
@@ -21,7 +21,7 @@ class APIKeyResource extends CrudResource<BaseModels.ApiKey.Model, ModelIDKey, A
     return super._getByID<BaseModels.ApiKey.Model>(id);
   }
 
-  public async create(workspaceID: string, body: Partial<BaseModels.ApiKey.Model>): Promise<BaseModels.ApiKey.Model & { APIKey: string }> {
```

Comment:
this might lead to some breaking type changes, but looking through the repos I can't find anywhere that actually consumes this.